src/js/models/v2/awardsV2/BaseAwardAmounts.js

```diff
@@ -154,4 +154,18 @@ const BaseAwardAmounts = {
     }
 };
 
+BaseAwardAmounts.getNonCombinedIdvAmounts = function getNonCombinedIdvAmounts(data) {
```

Comment:
Nice thanks man. Sounds good.

Reply:
@ebdabbs @jonhill13 

Doing this allows us to reuse the `get<awardSpendingCategory>Formatted` w/o having to rewrite them specifically for this case where we don't want the combined/aggregated IDV child/grandchild award data.

Want to know more about why we shouldn't overwrite private variables...

Reply:
I see what you are doing, and I do not see a way around it without updating a lot of other code. Hence, I approved the PR, and offered a suggestion.

Prefixing a property with `_` means treat this as private variable and should be treated as such, even though we can change them. (If we really wanted them to be private we should be using define property and set writable to false).  Private variable's should not change because we use those in the getter methods.  The properties that can change are the getter methods or if we want something different we should be adding new getter methods.

https://dev.to/somedood/emulating-private-variables-in-javascript-with-closures-and-factory-functions-2314